libs/libcommon/README.md

```diff
@@ -71,4 +71,7 @@ Set environment variables to configure the connection to S3.
 
 ## Rows Index
 
-- `ROWS_INDEX_MAX_ARROW_DATA_IN_MEMORY`: The maximun number of row groups to be loaded in memory.
\ No newline at end of file
+- `ROWS_INDEX_MAX_ARROW_DATA_IN_MEMORY`: The maximun number of row groups to be loaded in memory.
```

Comment:
I don't get what is the difference? surely format

btw, a nit:

maximun -> maximum